_posts/2017-10-05-apprenticeship-at-8thlight.md

```diff
@@ -0,0 +1,213 @@
+---
+layout: post
+title:  "Apprenticeship at 8th Light"
+date: 2017-10-05 14:23:00 +0200
+categories: how-we-learn 
+author_name: Wolfram Kriesing
+author_url : /author/wolframkriesing
+author_avatar: wolframkriesing
+show_avatar : true
+read_time : 20
+show_related_posts: false
+square_related: recommend-wolf
+feature_image: posts/2017-10-05-8thlight-2/structured-design.jpg
+---
+
+Officially it was 2006 that 8th Light says they had started the [apprenticeship program]. From reading a bit around and searching for related articles online, it seems that there had been ocurrences of it before. Jake Scruggs [writes][jake-writes]  in 2009 about his apprenticeship that he had started 5 years before in 2004 at Object Mentor, kind of the predecessor of 8th Light. But that’s lot to cover, something for another history post :).
+
+[jake-writes]: http://jakescruggs.blogspot.de/2009/04/third-day-of-craftsman-swap.html
+[apprenticeship program]: https://8thlight.com/apprenticeship
+
+## Disclaimer
+
+This post is describing the apprenticeship program of 8th Light as seen from the outside. I had not been in it. My colleague Tobias Pflug and me we had visited 8th Light in London last week to learn from them. So this post is just a summary of how I understand that 8th Light runs their apprenticeship program, it might contain errors. If you find any, please point them out to us.
```

Comment:
since i finally got my act together putting a blog post online I guess you can link me? ;}

Reply:
`I had not been in it` AFAIK this is the Past Perfect which means an action that happened in the past and which does not reflect any continuation in the present. Since you still haven't done an apprenticeship program, I guess using Past Perfect is wrong. Therefore I would use Present Perfect `I have not been in it` (or even Present Perfect Progressive, but I am unable to formulate this)

`we had visited`, why do you use the Past Perfect here? Usually it's used when two things happened in the past and you want to give them a chronology relation (X happened before Y, but both are in the past). Since you do not have this here, I would use the Simple Past `Tobias Pflug and me visited 8th Light...`